query-advisor/alerts.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Track Query Advisor Insights With Alerts'
+backlink_href: /docs/query-advisor
+backlink_title: 'Query Advisor'
+---
+
+Query Advisor automatically detects optimization opportunities, but you need to configure alerts to get notified when new insights are discovered. This guide covers setting up alerts for Query Advisor insights and integrating with your notification workflows.
+
+## Alert configuration overview
+
+Query Advisor checks are **enabled by default** for all databases in your pganalyze organization, but **alerts must be explicitly configured** through alert policies to receive notifications about newly detected optimization opportunities.
```

Comment:
Should we mention about plans? Technically it's not enabled by default for all databases, and we actually have quite a lot of customers with legacy plans.